src/js/components/DescriptionList/DefinitionRow.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Icon } from '../Icon/Icon'
+
+function DefinitionRow({ term, icon, children, className }) {
+  return (
+    <div className="flex gap-2">
+      <dt className="shrink-0 w-1/6 items-start text-xs font-medium text-gray-500 tracking-wider break-words">
+        {term}
+      </dt>
+      <dd className={`${className !== undefined ? className : ''}`}>
+        {icon && <Icon icon={icon} className="mr-2 " />}
```

Comment:
NIT .... extra space
```suggestion
        {icon && <Icon icon={icon} className="mr-2" />}
```